rustworkx-core/fuzz/fuzz_targets/test_fuzz_centrality.rs

```diff
@@ -0,0 +1,49 @@
+#![no_main]
+
+use arbitrary::{Arbitrary, Unstructured};
+use libfuzzer_sys::fuzz_target;
+use rustworkx_core::centrality::closeness_centrality;
+use rustworkx_core::petgraph::graph::UnGraph;
+
+#[derive(Debug, Arbitrary)]
+struct CentralityFuzzInput {
+    edges: Vec<(usize, usize)>,
+    node_count: usize,
+}
+
+fuzz_target!(|data: &[u8]| {
+    if let Ok(input) = CentralityFuzzInput::arbitrary(&mut Unstructured::new(data)) {
+        fuzz_centrality(input);
+    }
+});
+
+fn fuzz_centrality(input: CentralityFuzzInput) {
+    if input.node_count == 0 || input.node_count > 1000 {
+        return;
+    }
+
+    let mut graph = UnGraph::<(), ()>::default();
+    let nodes: Vec<_> = (0..input.node_count).map(|_| graph.add_node(())).collect();
+
+    for (u, v) in input.edges {
+        if u < input.node_count && v < input.node_count {
+            graph.add_edge(nodes[u], nodes[v], ());
+        }
+    }
+
+    // Closeness with wf_improved = true
+    let c_seq_true = closeness_centrality(&graph, true, 200);
+    let c_par_true = closeness_centrality(&graph, true, 1);
+    assert_eq!(
+        c_seq_true, c_par_true,
+        "Mismatch in wf_improved=true between sequential and parallel"
+    );
+
+    // Closeness with wf_improved = false
+    let c_seq_false = closeness_centrality(&graph, false, 200);
+    let c_par_false = closeness_centrality(&graph, false, 1);
+    assert_eq!(
+        c_seq_false, c_par_false,
+        "Mismatch in wf_improved=false between sequential and parallel"
+    );
+}
```

Comment:
This is more of a nitpick, but I'd loop over an array with `[true, false]` parameters instead of repeating the calls. Not strictly necessary, but it is a nice touch